bin/config-yargs.js

```diff
@@ -88,7 +88,7 @@ module.exports = function(yargs) {
 			},
 			mode: {
 				type: "string",
-				choices: ["development", "production"],
+				choices: ["development", "production", "none"],
```

Comment:
Can't these rules be imported from webpack repo, this is duplication, maybe using https://github.com/webpack/webpack/blob/master/schemas/WebpackOptions.json

Reply:
We definitely could, it actually makes sense, but I would improve it using a separate PR. Here I am just restoring something that we missed somehow... thanks for your feedback

Reply:
Cool